repo.sh

```diff
@@ -20,83 +20,79 @@ fi
 # When you add new services should add them to both repos and ssh_repos
 # (or non_release_repos and non_release_ssh_repos if they are not part
 # of Open edX releases).
-repos=(
-    "https://github.com/openedx/course-discovery.git"
-    "https://github.com/openedx/credentials.git"
-    "https://github.com/openedx/cs_comments_service.git"
-    "https://github.com/openedx/ecommerce.git"
-    "https://github.com/edx/edx-e2e-tests.git"
-    "https://github.com/openedx/edx-notes-api.git"
-    "https://github.com/openedx/edx-platform.git"
-    "https://github.com/openedx/xqueue.git"
-    "https://github.com/openedx/edx-analytics-dashboard.git"
-    "https://github.com/openedx/frontend-app-gradebook.git"
-    "https://github.com/openedx/frontend-app-payment.git"
-    "https://github.com/openedx/frontend-app-publisher.git"
-    "https://github.com/openedx/edx-analytics-dashboard.git"
-    "https://github.com/openedx/edx-analytics-data-api.git"
+declare -A repos=(
+    ["https://github.com/openedx/course-discovery.git"]="open-release/olive.master"
```

Comment:
Why didn't you link discovery and ecomm to our Pearson Advance repos?. @alexjmpb